databricks-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/databricks_run.py

```diff
@@ -10,19 +14,21 @@ def main():
     parser.add_argument("--env", dest="env", type=str)
     parser.add_argument("--conf-source", dest="conf_source", type=str)
     parser.add_argument("--package-name", dest="package_name", type=str)
+    parser.add_argument("--nodes", dest="nodes", type=str)
 
     args = parser.parse_args()
     env = args.env
     conf_source = args.conf_source
     package_name = args.package_name
+    nodes = args.nodes
 
     # https://kb.databricks.com/notebooks/cmd-c-on-object-id-p0.html
     logging.getLogger("py4j.java_gateway").setLevel(logging.ERROR)
     logging.getLogger("py4j.py4j.clientserver").setLevel(logging.ERROR)
 
     configure_project(package_name)
     with KedroSession.create(env=env, conf_source=conf_source) as session:
-        session.run()
+        session.run(node_names=[node.strip() for node in nodes.split(",")])
```

Comment:
What if there are no nodes provided?

Reply:
With this particular style of config generation there is no instance in which there will be no nodes. But I should check for that to be sure. Thanks for pointing it out :)